omod/src/main/webapp/resources/htmlforms/vitals.xml

```diff
@@ -0,0 +1,329 @@
+<htmlform formUuid="a000cb34-9ec1-4344-a1c8-f692232f6edd"
+      formName="Vitals"
+      formEncounterType="68a71486-1a54-468f-ac3e-7091a9a79584"
+      formVersion="1.0"
+      formDescription="Vitals Form"
+      formAddMetadata="yes"
+      formUILocation="patientDashboard.visitActions"
+      formOrder="8"
+      formIcon="icon-medkit"
+      formShowIf="(visit.active || !visit.active)"
+      formDisplayStyle="Simple"
+      formLabel="Vitals Form">
+<style>
+    #calculated-bmi {
+    font-weight: bold;
+    font-size: 1.4em;
+    }
+
+    .encounter-summary-container #calculated-bmi {
+    font-size: 1em;
+    font-weight: normal;
+    }
+    #99030{
+    padding: 5px;
+    }
+
+    .htmlformform form input, form select, form textarea, form ul.select, .form input, .form select, .form textarea, .form ul.select {
+    background-color: #FBFBFB;
+    border: 1px solid #CEC9C9;
+    border-radius: 5px;
+    margin-bottom: 10px;
+    padding: 5px;
+    min-width: 20%;
+    }
+</style>
+
+<!-- only show BMI if patient is more than 5 years old (on the encounterDate) -->
+<includeIf velocityTest="$patient.getAge($encounter.getEncounterDatetime()) > 5">
+
+    <script type="text/javascript">
+        var calculateBmi = function(wt, ht) {
+        var bmi = null
+        if (wt &amp;&amp; ht) {
+        bmi = wt / ((ht/100) * (ht/100));
+        }
+        return bmi ;
+        }
+
+    </script>
+
+    <ifMode mode="ENTER">
+        <script type="text/javascript">
+
+            // functions to handle updating the bmi when in ENTER mode
+            var updateBmi = function() {
+            var wt = htmlForm.getValueIfLegal('weight.value');
+            var ht = htmlForm.getValueIfLegal('height.value');
+
+            var bmi = calculateBmi(wt, ht);
+
+            if (bmi == null || isNaN(bmi)) {
+            jq('#calculated-bmi-wrapper').hide();
+            } else {
+            jq('#no-calculated-bmi').hide();
+            jq('#calculated-bmi-wrapper').show();
+            jq('#calculated-bmi').html(bmi.toFixed(1));
+            jq('#hidden-calculated-bmi').val(bmi.toFixed(1));
+            }
+            };
+
+            jq(function() {
+            jq('#calculated-bmi-wrapper').hide();
+
+            getField('weight.value').change(updateBmi)
+            getField('height.value').change(updateBmi);
+
+            updateBmi();
+            });
+
+            jq(function() {
+            jq('#bp_systolic input[type=text]').attr('min', '50');
+            });
+
+            jq(function() {
+            jq('#bp_diastolic input[type=text]').attr('min', '30');
+            });
+
+        </script>
+    </ifMode>
+
```

Comment:
@semujju Add code to redirect to simple UI 

<ifMode mode="EDIT">
        <script type="text/javascript">
            if (window.location.href.indexOf('editHtmlFormWithStandardUi') !== -1) {
                window.location.href = window.location.href.replace('editHtmlFormWithStandardUi','editHtmlFormWithSimpleUi');
            }
        </script>
    </ifMode>